zerver/webhooks/trello/view/card_actions.py

```diff
@@ -169,6 +181,21 @@ def get_changed_due_date_body(payload, action_type):
     }
     return fill_appropriate_message_content(payload, action_type, data)
 
+def get_managed_desc_body(payload, action_type):
+    # type: (Mapping[str, Any], Text) -> Text
+    data = {
+        'desc': prettify_date(get_action_data(payload)['card'].get('desc'))
```

Comment:
@MarcoMatarazzo `.get()` can return `None`, so in this case `pretiffy_date` takes `None` and raises an error `None has not method replace` - its not so easy to investigate. Better to see `KeyError`e